packages/terra-brand-footer/src/BrandFooter.module.scss

```diff
@@ -83,24 +83,24 @@
     .list-header + .menu::before {
       content: '';
       padding-left: 0;
-      padding-right: var(--terra-brand-footer-horizontal-nav-first-link-padding-before-right, 0.7142857142857143rem);
+      padding-right: var(--terra-brand-footer-horizontal-nav-first-link-before-padding-right, 0.7142857142857143rem);
     }
     /* stylelint-enable selector-max-compound-selectors */
 
     .separator {
       color: var(--terra-brand-footer-horizontal-nav-link-divider-color, #8baaca);
```

Comment:
This guy is `divider` vs `separator` below

Reply:
Fixed here https://github.com/cerner/terra-framework/pull/439/commits/3152c95dd5c9c39f028a92d2d2e884549d282331